### PR TITLE
Partsn sectionsn nav

### PR DESCRIPTION
**Pull request by WorkAccountLB**
2 new pages + updated styles + large modification of crimcodegridlist.js

param to pass from Section page to content page is  "heading2key".

### CODE REVIEW

File: src/components/CrimCodeGridList.js [hunk 1]
```diff
@@ -45,4 +42,34 @@ const CrimCodeGridList = props => {
...
+              </Text>
+              <Text style={styles.heading2TitleText}>
+                {props.heading2titletext}
+              </Text>
+            </View>
+
+            <Icon
+              style={styles.partsIcon}
+              name={'arrow-forward-ios'}
+              size={18}
```
Comment (doolaik) [inline src/components/CrimCodeGridList.js:67]: Can we decide on the size of these Icons? I have 20 up top, and 18 here. 20 on LegislationGridList too.

File: src/navigation/StackNavigation.js [hunk 2]
```diff
@@ -26,6 +28,11 @@ const ContentStack = () => {
         name="BrowseCCScreen"
         component={BrowseCCScreen}
       />
+      <ContentNavigator.Screen name="PartsCCScreen" component={PartsCCScreen} />
```
Comment (doolaik) [inline src/navigation/StackNavigation.js:31]: With these 2 new screens, does it replace BrowseCCScreen? If so, can you just make a comment that those need to be removed from here.

File: src/components/CrimCodeGridList.js [hunk 3]
```diff
@@ -9,31 +9,28 @@ import Icon from 'react-native-vector-icons/MaterialIcons';
 //USER IMPORTS
 import styles, {colors} from '../assets/styles';
 
-const CrimCodeGridList = props => {
+export const CrimCodePartsList = props => {
```
Comment (doolaik) [inline src/components/CrimCodeGridList.js:12]: Why can't this component be genericized to be used for both PartsCCScreen and SectionCCScreen? 

These 2 are the exact same code except for a few different prop names; make the key names the same and pass them into the component you've created.

File: src/screens/PartsCCScreen.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          setDistinctPartsList(temp);
+        },
+      );
+    });
+  };
+
+  const renderList = itemdata => {
+    return (
+      <View>
+        <CrimCodePartsList
```
Comment (doolaik) [inline src/screens/PartsCCScreen.js:43]: Regarding my comment on genericizing the CrimCodeGridList - Like if this was still:
```          
<CrimCodeGridList
    heading1key={itemdata.item.heading1key}
    heading2key={itemdata.item.heading2key}
    heading1label={itemdata.item.heading1label}
    heading1titletext={itemdata.item.heading1titletext}
    firstSectionLabel={itemdata.item.firstSectionLabel}
    heading2titletext={itemdata.item.heading2titletext}
  />;
```
I think even if a prop is missing, it'll still render so for your parts, you can still send it as:
```
<CrimCodeGridList
          heading1key={itemdata.item.heading1key}
          heading1label={itemdata.item.heading1label}
          heading1titletext={itemdata.item.heading1titletext}
        />
```

File: src/assets/styles.js [hunk 5]
```diff
@@ -130,6 +130,56 @@ export default StyleSheet.create({
   CCcontent: {
     height: 600,
   },
+  //Criminal Code PARTS/SECTION Screen Styling - Sprint 4
```
Comment (doolaik) [inline src/assets/styles.js:133]: I'm gonna feel bad for our designer that we will dump this on 😄 

There's probably so much overlapping styling that we've all put in that could be cleaned up 🤣

File: src/navigation/StackNavigation.js (hunk 2 above)
Comment (WorkAccountLB) [reply src/navigation/StackNavigation.js:31]: Yes it does replace BrowseCCScreen.. yes I can put a TODO to delete BrowseCCScreen.  Doing it now.

File: src/components/CrimCodeGridList.js (hunk 3 above)
Comment (WorkAccountLB) [reply src/components/CrimCodeGridList.js:12]: I could do that.. but the styling is slightly different.  But I will proceed as you suggest.

File: src/components/CrimCodeGridList.js [hunk 6]
```diff
@@ -9,31 +9,28 @@ import Icon from 'react-native-vector-icons/MaterialIcons';
...
-                  styles.sentenceWrap,
-                  {marginLeft: 10, color: colors.primaryText},
-                ]}>
-                {props.sectionHeader}
+              <Text style={styles.heading1TitleText}>
+                {props.heading1titletext}
               </Text>
             </View>
 
             <Icon
```
[the remark this answers is not in the document]
Comment (WorkAccountLB) [reply src/components/CrimCodeGridList.js:32]: Truth.  Do you want at as an Asset component?

File: src/assets/styles.js (hunk 5 above)
Comment (WorkAccountLB) [reply src/assets/styles.js:133]: Absolutely!  I am trying to remove those I have done previously but I think I missed a couple.  That's why I decided to delineate as other have done with comments.  And I scour the file for things I can reuse as much as I can.

File: src/components/CrimCodeGridList.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (doolaik) [reply src/components/CrimCodeGridList.js:32]: Yea, I'd consider these kind of things as assets.

File: src/screens/PartsCCScreen.js (hunk 4 above)
Comment (WorkAccountLB) [reply src/screens/PartsCCScreen.js:43]: You are right.  I will work on generesizing it right now.  This issue I have to figure out is the navigation.  I'll have to do if statements to check to see which was the previous screen so the navigation to the next screen is correct.

File: src/components/CrimCodeGridList.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (WorkAccountLB) [reply src/components/CrimCodeGridList.js:32]: DOne

File: src/screens/PartsCCScreen.js (hunk 4 above)
Comment (WorkAccountLB) [reply src/screens/PartsCCScreen.js:43]: Done

File: src/components/CrimCodeGridList.js (hunk 1 above)
Comment (WorkAccountLB) [reply src/components/CrimCodeGridList.js:67]: Fixed.

**Review by doolaik — APPROVED** [review]
Good job on genericizing CrimCodeGridList

**Pull request merged**